twenty-first/src/shared_math/traits.rs

```diff
@@ -16,9 +16,17 @@ where
 // TODO: Assert if we're risking inverting 0 at any point.
 pub trait Inverse
 where
-    Self: Sized,
+    Self: Sized + Zero,
```

Comment:
I'm not sure this is a breaking change, requiring a version bump of “major.” Let me know.

Reply:
Since all `impl Inverse` are for types that have `Zero`, we won't experience it that way, but hypothetical subscribers to the trait alone would experience it as breaking, yes. I'd mark this as breaking, but also: You don't need to release a new version of twenty-first yet because of this. We have a handful of small changes we'd like to push as a result of merging #25 onto other codebases. Let's schedule a version bump.